packages/elements/src/pagination/__test__/pagination.test.js

```diff
@@ -6,21 +6,29 @@ import {
   oneEvent,
   triggerFocusFor,
   keyboardEvent,
-  aTimeout
+  aTimeout,
+  waitUntil
 } from '@refinitiv-ui/test-helpers';
 
+// Translations polyfills
+import '@formatjs/intl-locale/polyfill.iife';
+import '@formatjs/intl-getcanonicallocales/polyfill.iife';
+import '@formatjs/intl-pluralrules/polyfill.iife';
+import '@formatjs/intl-pluralrules/locale-data/en';
+import '@formatjs/intl-pluralrules/locale-data/zh';
```

Comment:
There are no tests for languages apart of English. You do not need to include all locales.

To make tests consistent across all locales, please provide `lang` for every case.

Reply:
Do you mean to provide the `lang="en-gb"` for every translation test cases only?

Reply:
It should import these (as we test English only):

```javascript
import '@formatjs/intl-getcanonicallocales/polyfill.iife';
import '@formatjs/intl-pluralrules/polyfill.iife';
import '@formatjs/intl-pluralrules/locale-data/en';
```

Every sample must enclude lang, e.g.: `<ef-pagination lang="en-gb"></ef-pagination>`

Reply:
Fixed